CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 ## Unreleased
 
+FEATURES:
+* Support Kubernetes health probe synchronization with Consul for connect injected pods via `connectInject.healthChecks` [[GH-651](https://github.com/hashicorp/consul-helm/pull/651)].
+  The default behavior for this feature is `enabled: true`.
+  See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
+  **It is recommended to enable TLS with this setting enabled.**
+
+BUG FIXES:
+* Support new versions of timeout in ent-license job [[GH-654](https://github.com/hashicorp/consul-helm/pull/654)].
```

Comment:
Let's be explicit about the bug that was fixed. Something like "Fix issue where Consul enterprise license job would fail for Consul versions >= 1.8.1".

Also we should link to the issue fixed for bug fixes if there is one (https://github.com/hashicorp/consul-helm/issues/647)